config/application.rb

```diff
@@ -58,8 +58,7 @@ class Application < Rails::Application
     config.active_job.queue_adapter = :good_job
 
     FileUtils.mkdir_p(Rails.root.join('log'))
```

Comment:
was this `mkdir_p` for that log below? Could we remove it maybe?
```suggestion
```

Reply:
Turns out detaching the other log subscriber is probably the way to go rather than removing the Logger entirely.